tests/components/guardian/test_config_flow.py

```diff
@@ -54,7 +70,7 @@ async def test_step_user(hass, ping_client):
         DOMAIN, context={"source": SOURCE_USER}, data=conf
     )
     assert result["type"] == data_entry_flow.RESULT_TYPE_CREATE_ENTRY
-    assert result["title"] == "Elexa Guardian (192.168.1.100)"
+    assert result["title"] == "ABCDEF123456"
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use of assert detected. The enclosed code will be removed when compiling to optimised byte code.](https://app.codacy.com/manual/foreign-sub/home-assistant/pullRequest?prid=5596821)